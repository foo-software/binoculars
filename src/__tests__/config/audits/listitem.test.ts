import auditListitem from '../../../config/audits/listitem';

describe('listitem audit', () => {
  it('should match snapshot', () => {
    expect(auditListitem('en')).toMatchSnapshot();
  });
});
