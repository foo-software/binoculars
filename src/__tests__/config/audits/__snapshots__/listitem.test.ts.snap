// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`listitem audit should match snapshot 1`] = `
Object {
  "description": "Screen readers require list items (\`<li>\`) to be contained within a parent \`<ul>\` or \`<ol>\` to be announced properly. [Learn more](https://web.dev/listitem/).",
  "title": "List items (\`<li>\`) are contained within \`<ul>\` or \`<ol>\` parent elements",
}
`;
